week_1/2_apr/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,100 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.left-section {
+    display: flex;
+    align-items: center;
+}
+
+.hamburger-menu {
+    height: 24px;
+    margin-left: 24px;
+    margin-right: 20px;
+}
+
+.youtube-logo {
+    height: 22px;
+}
+
+.middle-section {
+    flex: 1;
+    margin-left: 70px;
+    margin-right: 35px;
+    max-width: 400px;
+    display: flex;
+    align-items: center;
+}
+
+.search-bar {
+    flex: 1;
```

Comment:
What is use of flex:1? why we use it

Reply:
@abhi-codal I did it to make the search bar flexible based on the window size. If we give one item flex: 2, and another item flex: 1, then the first item will take two times the amount of space than another item in the axis.